internal/core/window.rs

```diff
@@ -645,6 +666,9 @@ impl WindowInner {
     /// If the component's root item is a Window element, then this function synchronizes its properties, such as the title
     /// for example, with the properties known to the windowing system.
     pub fn update_window_properties(&self) {
+        let window_adapter = self.window_adapter();
+        let Some(window_adapter) = window_adapter.internal(crate::InternalToken) else {return};
```

Comment:
Maybe it would be worth having a `window_adapter_internal()` function on `WindowInner` to have less spread of the `InternalToken`?